qutebrowser/utils/log.py

```diff
@@ -175,6 +176,10 @@ def init_log(args):
     _log_inited = True
 
 
+def change(filters):
+    console.addFilter(LogFilter(filters.split(',')))
```

Comment:
Instead of adding a new filter (which means I won't be able to change the filter I set on the command line, only add new filters), you could get the existing filter (e.g. by assigning it to a `console_filter` global when it's created in `init_log`), and set the `_names` attribute there (after renaming it to `names` to make it "public").